src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/ConsoleApp/GenerateSwaggerConsoleApp.cs

```diff
@@ -0,0 +1,53 @@
+using System.Threading.Tasks;
+using Cocona;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Extension;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Interface;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Enums;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.ConsoleApp
+{
+    public class GenerateSwaggerConsoleApp
+    {
+        /// <summary>
+        ///     Generates the OpenAPI document.
+        /// </summary>
+        /// <param name="project">Project path.</param>
+        /// <param name="apibaseurl"></param>
+        /// <param name="configuration">Compile configuration.</param>
+        /// <param name="target"></param>
+        /// <param name="version">OpenAPI version.</param>
+        /// <param name="format">OpenAPI output format.</param>
+        /// <param name="output">Output path.</param>
+        public async Task Run(
+            [FromService] ICustomApiMockCreator customApiMockCreator,
+            [FromService] ICustomOpenApiCreator customOpenApiCreator,
+            [FromService] ICustomOpenApiWriter customOpenApiWriter,
+            [Option('p', Description = "Api Project path. Default is current directory")]
+            string project = ".",
+            [Option('a', Description = "ApiBaseUrl. Default is 'localhost'")]
+            string apibaseurl = "localhost",
+            [Option('c', Description = "Configuration. Default is 'Debug'")]
+            string configuration = "Debug",
+            [Option('t', Description = "Target framework. Default is 'net5.0'")]
+            string target = "net5.0",
```

Comment:
Let's set the default value of `net6.0` as it's LTS.

Reply:
Changed to net6.0 in new commit